src/commands/list_drive.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-console */
+import { arDriveFactory, cliArweave, cliWalletDao } from '..';
+import { ArDriveAnonymous } from '../ardrive';
+import { ArFSDAOAnonymous, ArFSPrivateFileOrFolderWithPaths, ArFSPublicFileOrFolderWithPaths } from '../arfsdao';
+import { CLICommand, ParametersHelper } from '../CLICommand';
+import { DriveIdParameter, SeedPhraseParameter, WalletFileParameter } from '../parameter_declarations';
+import { alphabeticalOrder } from '../utils/sort_functions';
+
+new CLICommand({
+	name: 'list-drive',
+	parameters: [DriveIdParameter, SeedPhraseParameter, WalletFileParameter],
+	async action(options) {
+		const parameters = new ParametersHelper(options, cliWalletDao);
+		const driveId = parameters.getRequiredParameterValue(DriveIdParameter);
+		let children: (ArFSPrivateFileOrFolderWithPaths | ArFSPublicFileOrFolderWithPaths)[];
+		const maxDepth = 0;
```

Comment:
Why is the default behavior to list a max depth of 0? As per the manpage:
```
list-drive:
  • Gets all ArFS Metadata and Txids for the files and folders within this drive
  --drive-id: the ArFS drive ID for the drive
  --drive-password: the drive password for drive of the folder identified by --drive-id
  • Required only for folders residing in private drives
  • Can NOT be used in conjunction with --drive-key
  --drive-key: the drive key for the parent drive of the folder identified by --folder-id
  • Required only for folders residing in private drives
  • Can NOT be used in conjunction with --drive-password
  --get-all-revisions: (OPTIONAL) gets every revision of each ArFS entity within this drive
```
most notably `Gets all ArFS Metadata and Txids for the files and folders within this drive`, key word `all`. So I believe the max depth value here should be Number.MAX_SAFE_INTEGER (or Number.POSITIVE_INFINITY).